### PR TITLE
Missing code style suggestions from Carriage Return fix PR

### PR DESCRIPTION
**Pull request by dalthviz**
Follow up for #607 as discussed at https://github.com/jupyter/qtconsole/pull/607#issuecomment-2276530999

### CODE REVIEW

File: qtconsole/console_widget.py [hunk 1]
```diff
@@ -1694,9 +1694,12 @@ def _on_flush_pending_stream_timer(self):
         self._flush_pending_stream()
 
     def _flush_pending_stream(self):
-        """ Flush pending text into the widget. Only applies to text that is pending
-            when the console is in the running state. Text printed when console is
-            not running is shown immediately, and does not wait to be flushed.
+        """
+        Flush pending text into the widget.
+
+        Only applies to text that is pending when the console is in the
```
Comment (ccordoba12) [inline qtconsole/console_widget.py:1700]:
```suggestion
        It only applies to text that is pending when the console is in the
```

File: qtconsole/console_widget.py [hunk 2]
```diff
@@ -1017,15 +1017,15 @@ def _append_custom(self, insert, input, before_prompt=False, *args, **kwargs):
                     and self._append_before_prompt_pos != self._get_end_pos():
                 cursor.setPosition(self._append_before_prompt_pos)
 
-                # If we appending on the same line as the prompt, use insert mode
+                # If we're appending on the same line as the prompt, use insert mode
```
Comment (ccordoba12) [inline qtconsole/console_widget.py:1020]:
```suggestion
                # If we're appending on the same line as the prompt, use insert mode.
```

**Review by ccordoba12 — COMMENTED** [review]
Two tiny suggestions that I missed in my previous review, the rest looks good to me.

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz!

**Comment by TheMatt2** [comment]
I am total on board with these changes. Thanks for taking care of this. I've been busy with other things.

**Pull request merged**